conf/es/spain03.yml

```diff
@@ -463,6 +468,21 @@ years:
     names:
       en: Asunción
       es: Asunción
+   - public_holiday: true
+    date: '2021-08-16'
+    names:
+      en: Asunción in Lieu
```

Comment:
I would have expected to see "in lieu" (i.e. no capital "L"), because it's a modifying term not a part of the title for the day.